packages/flutter_tools/lib/src/build_system/targets/dart.dart

```diff
@@ -102,7 +102,7 @@ class CopyFlutterBundle extends Target {
     environment.outputDir.createSync(recursive: true);
 
     // Only copy the prebuilt runtimes and kernel blob in debug mode.
-    if (buildMode == BuildMode.debug) {
+    if (buildMode == BuildMode.debug || buildMode == BuildMode.jitRelease) {
```

Comment:
Consider elevating `BuildMode` from an enum to a class (or adding a class that has the enum as a field), and moving the logic here and below to that class.